### ISSUE TITLE
Hellhounds bugged

### ISSUE THREAD

**Issue by ilucky1995** [TheTurkeyDev/Call_Of_Minecraft-Zombies#125]
They spawn everywhere in the arena(including areas not unlocked) and they also don't go to players to attack them, they are just running around like headless chickens.

Not sure if there is a specific spawn point i need to set for them or if i did something wrong.

**Comment by TheTurkeyDev** [comment]
> They spawn everywhere in the arena(including areas not unlocked)
Unfortunately Hell hound spawn logic just isn't perfect and is very difficult to do. #22 Is already "tracking" this.

>  They also don't go to players to attack them, they are just running around like headless chickens.
I haven't heard this before. Something doesn't seem right to me. Do the regular zombies work? They should have the same pathing

>And they just die automatically if you don't kill them yourself.
This is be all, end all fix to this overall issue. It's there so that worst case scenario and you can't get to a wolf, the game will still continue.

**Comment by ilucky1995** [comment]
> > They spawn everywhere in the arena(including areas not unlocked)
> > Unfortunately Hell hound spawn logic just isn't perfect and is very difficult to do. #22 Is already "tracking" this.
> 
> > They also don't go to players to attack them, they are just running around like headless chickens.
> > I haven't heard this before. Something doesn't seem right to me. Do the regular zombies work? They should have the same pathing
> 
> > And they just die automatically if you don't kill them yourself.
> > This is be all, end all fix to this overall issue. It's there so that worst case scenario and you can't get to a wolf, the game will still continue.

Yes normal zombies do work fine and will go towards the players, just not the hellhounds(i mean they do sometimes but not always, and if they do it's in a really weird way) and there seem to be a delay in them attacking you if one managed to finally reach you.
